command/v7/shared/app_summary_displayer_test.go

```diff
@@ -702,13 +708,17 @@ var _ = Describe("app summary displayer", func() {
 								Strategy:         constant.DeploymentStrategyRolling,
 								StatusValue:      constant.DeploymentStatusValueActive,
 								StatusReason:     constant.DeploymentStatusReasonCanceling,
-								LastStatusChange: "2024-07-29T17:32:29Z",
+								LastStatusChange: LastStatusChangeTimeString,
 							},
 						}
 					})
 
 					It("displays the message", func() {
-						Expect(testUI.Out).To(Say(`Rolling deployment currently CANCELING \(since Mon 29 Jul 13:32:29 EDT 2024\)`))
+						parsedTime, err := time.Parse(time.RFC3339, LastStatusChangeTimeString)
+						Expect(err).To(Not(HaveOccurred()))
+
+						timeString := parsedTime.Local().Format("Mon 02 Jan 15:04:05 MST 2006")
```

Comment:
maybe we should extract the format string here for clarity

Reply:
Good catch